### ISSUE TITLE
Webhook on meeting starts

### ISSUE THREAD

**Issue by raelix** [calcom/cal.com#12728]
### Is your proposal related to a problem?
No, it isn't but would be nice to have the ability of calling a webhook at the start time of a meeting.

### Describe the solution you'd like
The solution aims to have one more available option for webooks. It would allow to call a webhook at the start time of a meeting.

##### House rules
- If this issue has a `🚨 needs approval` label, don't start coding yet. Wait until a core member approves feature request by removing this label, then you can start coding.
  - For clarity: Non-core member issues automatically get the `🚨 needs approval` label.
  - Your feature ideas are invaluable to us! However, they undergo review to ensure alignment with the product's direction.

**Comment by sakshambp** [comment]
can you give more context on what it means?

**Comment by raelix** [comment]
When this will be available on the cloud cal.com?

**Comment by hrshkshri** [comment]
@PeerRich 
I think this issue is resolved ...

**Comment by Arif3028** [comment]
is the issue still exits?

**Comment by TusharBhatt1** [comment]
The PR has been merged , bounty awarded , its seems the issue is resolved.
